File: android/app/src/main/java/com/yourcompany/attendance_registration/MyDeviceAdminReceiver.java
package com.yourcompany.attendance_registration;

import android.app.admin.DeviceAdminReceiver;
import android.content.Context;
import android.content.Intent;
import android.widget.Toast;

public class MyDeviceAdminReceiver extends DeviceAdminReceiver {
    void showToast(Context context, String msg) {
        Toast.makeText(context, msg, Toast.LENGTH_SHORT).show();
    }

    @Override
    public void onEnabled(Context context, Intent intent) {
        showToast(context, "Device Admin: enabled");
    }

    @Override
    public void onDisabled(Context context, Intent intent) {
        showToast(context, "Device Admin: disabled");
    }
}

File: android/app/src/main/kotlin/com/example/attendance_registration/MyDeviceAdminReceiver.kt
package com.example.attendance_registration

import android.app.admin.DeviceAdminReceiver
import android.content.Context
import android.content.Intent
import android.widget.Toast

class MyDeviceAdminReceiver : DeviceAdminReceiver() {
    private fun showToast(context: Context, msg: String) {
        Toast.makeText(context, msg, Toast.LENGTH_SHORT).show()
    }

    override fun onEnabled(context: Context, intent: Intent) {
        showToast(context, "Device Admin: enabled")
    }

    override fun onDisabled(context: Context, intent: Intent) {
        showToast(context, "Device Admin: disabled")
    }
}
